_schemaV2.graphql

```diff
@@ -7341,6 +7347,9 @@ type Query {
   # Find PartnerStats
   analyticsPartnerStats(partnerId: String!): AnalyticsPartnerStats
 
+  # Query UserStats
+  analyticsUserStats(userId: String!): AnalyticsUserStats
```

Comment:
**nitpick**: `userId` -> `userID`

I'm noticing some inconsistencies in places with this 😅. Compairing `parterId` vs `internalID`.

Reply:
Discussed offline -- we will be leaving it as is for now!